tmt-app/src/Components/CustomCard.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Fragment } from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
+import { Link } from "react-router-dom";
+
+function CustomCard({ dummyData }) {
```

Comment:
Can we rename dummy data so it makes sense for this component?

Is there also a component name that can provide more context to what this component is and what it does?